custom_components/hyxi_cloud/coordinator.py

```
@@ -89,6 +112,33 @@ def __init__(self, hass: HomeAssistant, client: HyxiApiClient, entry: ConfigEntr
+                _LOGGER.debug(
+                    "Pre-seeding coordinator from cache (%d devices) before first API call",
+                    len(devices),
+                )
+                self.data = devices  # pylint: disable=attribute-defined-outside-init
+                self.hyxi_metadata["api_status"] = "Starting (cached)"
+            elif devices:
+                _LOGGER.debug(
+                    "Cache found but expired (>%d days old), skipping pre-seed",
+                    CACHE_MAX_AGE.days,
+                )
+        except Exception:  # pylint: disable=broad-except
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)